server/Dockerfile

```diff
@@ -26,4 +26,10 @@ WORKDIR /code/server
 # -------------------------------------
 COPY scripts_startup/azure_dev.dpcreator.org.sh /usr/bin/azure_dev.dpcreator.org.sh
 COPY scripts_startup/azure_demo.dpcreator.org.sh /usr/bin/azure_demo.dpcreator.org.sh
-RUN chmod u+x /usr/bin/*.sh
\ No newline at end of file
+RUN chmod u+x /usr/bin/*.sh
+
+# -------------------------------------
+# Install libmagic
+# to determine file types
+# -------------------------------------
+RUN apt-get install libmagic1
```

Comment:
Please move to after line 5 -- e.g. so it's further up the in docker layers, not re-installed after each code change